nixos/tests/gerrit.nix

```diff
@@ -25,11 +19,9 @@ in {
           listenAddress = "[::]:80";
           jvmHeapLimit = "1g";
 
-          plugins = [ lfs ];
           builtinPlugins = [ "hooks" "webhooks" ];
           settings = {
             gerrit.canonicalWebUrl = "http://server";
-            lfs.plugin = "lfs";
             plugins.allowRemoteAdmin = true;
```

Comment:
It might be worth dropping that line if the test doesn't test the plugins

Reply:
Done